sessionctx/variable/session_test.go

```diff
@@ -88,9 +98,12 @@ func (*testSessionSuite) TestSlowLogFormat(c *C) {
 # Is_internal: true
 # Digest: 42a1c8aae6f133e934d4bf0147491709a8812ea05ff8819ec522780fe657b772
 # Stats: t1:pseudo
+# Num_cop_tasks: 10
+# Cop_process: Avg_time: 1s P90_time: 2s Max_time: 3s
+# Cop_wait: Avg_time: 10ms P90_time: 20ms Max_time: 30ms
```

Comment:
Remember to fix this format in other PR.

Reply:
Yes, in order to avoid conflict, I will cherry-pick the other PR which fix this format after this PR is merged.